lib/note-detail/toggle-task/constants.ts

```diff
@@ -3,6 +3,6 @@ export const selectors = {
   markdownRoot: '[data-markdown-root]',
 };
 
-export const taskPrefixRegex = /^(\s*)(-[ \t]+\[[xX\s]?\])/g;
+export const taskPrefixRegex = /^(\s*)(-[ \t]+\[[xX\s]?\])/gm;
```

Comment:
@mirka can you remember if there was any specific reason not to include all search results here by adding the `m`? I didn't see any mention of it in the commit log but I don't want to accidentally introduce a bug you intentionally avoided.

Reply:
It seems like between the time I added this regex and now, the behavior of the `Comfy`/`Expanded` note list views have changed. Before, it showed the title and only the first line of the body, with `Expanded` showing more of the first line if it was a long paragraph. Now, it's concatenating multiple lines of the body if the lines are short.

So back then we didn't need the `m` flag, simply because a note body preview would never show more than one task item line. (In that sense, it's actually the `g` flag that was superfluous!)

Reply:
ah! that makes sense. we did update the preview generation so that it would show the start of the first few lines of the note instead of collapsing them. we missed the impact here then when we made that change.

thanks for chiming in and helping @mirka!